ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -830,6 +832,17 @@ internal class CustomerCenterViewModelImpl(
                 null
             }
 
+            val originalAppUserId = latestCustomerInfo?.originalAppUserId ?: ""
+            val originalPurchaseDate = latestCustomerInfo?.originalPurchaseDate?.let {
+                dateFormatter.format(
+                    it,
+                    locale,
+                )
+            }
+            val shouldShowUserDetailsSection =
+                customerCenterConfigData.support.displayUserDetailsSection &&
+                    (originalAppUserId.isNotBlank() || originalPurchaseDate != null)
```

Comment:
One thing... Would we still want to show the account details section even if `originalPurchaseDate` is null? Accessing the userId could still be useful I think even if the user never made any purchases, so I'm not sure if we should tie it to that.

Reply:
I am matching iOS implementation. We've been going back and forth with this, so the dashboard flag just hides it entirely. 😓 

1. The purchase date is always present
2. User id depends on DEBUG. 

This PR just adds a dashboard controlled flag to hide the section entirely.